addons/binding/org.openhab.binding.sensebox/README.md

```diff
@@ -0,0 +1,114 @@
+| measurements#pressure                | Air pressure                                                 | Number              | False    |
+| measurements#humidity                | Humidity                                                     | Number              | False    |
+| measurements#temperature             | Temperature                                                  | Number              | False    |
+| lastReported#uvIntensityLastReported | The timestamp when uv radiation intensity was last reported  | DateTime            | True     |
+| lastReported#luminanceLastReported   | The timestamp when illuminance was last reported             | DateTime            | True     |
+| lastReported#pressureLastReported    | The timestamp when pressure was last reported                | DateTime            | True     |
+| lastReported#humidityLastReported    | The timestamp when humidity was last reported                | DateTime            | True     |
+| lastReported#temperatureLastReported | The timestamp when temperature was last reported             | DateTime            | True     |
+
+The channels starting with "descriptors" are defined on the API server.
+
+## Channel configuration
+
+...
+
+## Full example
+
+sensebox.things:
+
+```
+Thing sensebox:box:zugspitze [ senseBoxId = "578cf2ccccff9d1000bd9198", refreshInterval = 900 ]
+```
+
+sensebox.items:
+
+```
+Location Zugspitze_Location                                                               (Zugspitze)                { channel="sensebox:box:zugspitze:descriptors#location" }
+
+String  Zugspitze_Image_URL        "Zugspitze Image"                                      (Zugspitze)                { channel="sensebox:box:zugspitze:descriptors#imageUrl" }
+
+Number  Zugspitze_Humidity         "Zugspitze Humidity [%.1f %%]"         <humidity>      (Zugspitze, Weather)       { channel="sensebox:box:zugspitze:measurements#humidity" }
+Number  Zugspitze_Luminance        "Zugspitze Light Level [%.1f lx]"      <light>         (Zugspitze, Weather)       { channel="sensebox:box:zugspitze:measurements#luminance" }
+Number  Zugspitze_Pressure         "Zugspitze Pressure [%.1f hPa]"        <pressure>      (Zugspitze, Weather)       { channel="sensebox:box:zugspitze:measurements#pressure" }
+Number  Zugspitze_Temperature      "Zugspitze Temperature [%.1f °C]"      <temperature>   (Zugspitze, Weather)       { channel="sensebox:box:zugspitze:measurements#temperature" }
+Number  Zugspitze_UVIntensity      "Zugspitze UvIntensity [%.1f μW/cm²]"  <light>         (Zugspitze, Weather)       { channel="sensebox:box:zugspitze:measurements#uvIntensity" }
+
+```
+
+sensebox.sitemap:
+
+```
+sitemap sensebox label="SenseBox Zugspitze" {
+	Text item=Zugspitze_Temperature
+	Text item=Zugspitze_Pressure
+	Text item=Zugspitze_Humidity
+	Text item=Zugspitze_Luminance
+	Text item=Zugspitze_UVIntensity
+	Mapview item=Zugspitze_Location height=10
+	Image item=Zugspitze_Image_URL
+}
+```
+
+## Reporting issues
```

Comment:
Please remove this section - the way to report issues should be the same for all bindings, so there should be no need to provide such sections for every single binding.